pom.xml

```diff
@@ -224,6 +224,14 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
forget to remove this dependency from pom. Will remove it after the Azure pipeline finished.